staging/src/k8s.io/apiextensions-apiserver/pkg/generated/openapi/zz_generated.openapi.go

```diff
@@ -19,8 +19,6 @@ limitations under the License.
 
 // Code generated by openapi-gen. DO NOT EDIT.
 
-// This file was autogenerated by openapi-gen. Do not edit it manually!
```

Comment:
Where did this go

Reply:
When I restructured how boilerplate is emitted, I did not preserve it.  Go has a convention, documented in `go help generate`:

```
To convey to humans and machine tools that code is generated,
generated source should have a line that matches the following
regular expression (in Go syntax):

        ^// Code generated .* DO NOT EDIT\.$
```